pkgs/by-name/na/nanopb/package.nix

```diff
@@ -0,0 +1,122 @@
+{ stdenvNoCC
+, callPackage
+, fetchFromGitHub
+, buildPackages
+, lib
+, enable_malloc ? false
```

Comment:
I think the convention is to use camelCase, not snake_case, for package options.